spec/unit/doc/defined-names.spec.js

```diff
@@ -70,15 +70,17 @@ describe('DefinedNames', function() {
   it('skips values with invalid range', function() {
     var dn = new DefinedNames();
     dn.model = [
-      {name: 'eq', ranges: ['"="']},
-      {name: 'ref', ranges: ['#REF!']},
-      {name: 'single', ranges: ['Sheet3!$A$1']},
-      {name: 'range', ranges: ['Sheet3!$A$2:$F$2228']}
+      {name: 'eq', ranges: ['"="'], formula: ''},
```

Comment:
input data in tests should not be modified, it causes lost possibility to test backward compatibility